src/app/core/components/header/header.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+
+  opened = false;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
Please remove unused constructor and ngOnInit

Reply:
fixed